Project/Source/GameObject.cpp

```diff
@@ -170,12 +174,6 @@ void GameObject::AddMask(MaskType mask_) {
 	case MaskType::PLAYER:
 		mask.bitMask |= static_cast<int>(mask_);
 		break;
-	case MaskType::CAST_SHADOWS:
```

Comment:
Is this deprecated code that is being removed?

Reply:
Yes, this was deprecated when merging shadows with Jordi.